docs/deployment.md

```diff
@@ -5,8 +5,10 @@ the [BDK Spring Boot integration](https://symphony-bdk-java.finos.org/spring-boo
 The [Getting started](./getting-started.md) guide explains how the Symphony Generator can be used to create a sample
 project with configuration files.
 
-_In its current version, the WDK does not persist state except in memory, so restarting the bot means all running
-workflow's state is lost. Running multiple instances of the bot for high availability is also not yet supported._
+By default, the WDK does not persist state except in memory, so restarting the bot means all running
```

Comment:
do we want to make the usage of the h2 file DB the default? (not for unit tests) so that is easier for users to upgrade a bot?

Reply:
That's what was specified in the ticket and I think it makes sense as one would only need to test the workflow in the beginning. The database will be needed once we test the workflow and know how it works in order to make real things.